src/shared/components/common/pictrs-image.tsx

```diff
@@ -21,6 +23,14 @@ export class PictrsImage extends Component<PictrsImageProps, any> {
   }
 
   render() {
+    let user_blur_nsfw = true;
+    if (UserService.Instance.myUserInfo) {
+      user_blur_nsfw =
+        UserService.Instance.myUserInfo?.local_user_view.local_user.blur_nsfw;
+    }
+
+    const blur_image = this.props.nsfw && user_blur_nsfw;
```

Comment:
`undefined` is falsy in JS/TS, so this can be simplified:

```suggestion
    const blur_image = this.props.nsfw && UserService.Instance.myUserInfo?.local_user_view.local_user.blur_nsfw;
```

Reply:
I agree this looks better, I had it default to `true`, but only logged-in users will be able to see NSFW, so if they are logged out, it's fine if the default is false 🤷‍♂️.

I think there should be some admin-settings at some point to control both logged-out NSFW visibility blur settings, but that's another story.